examples/CMakeLists.txt

```diff
@@ -26,4 +27,15 @@ foreach(s ${examples})
     get_filename_component (sn ${s} NAME_WE)
     add_executable(${sn} ${s})
     target_link_libraries(${sn} ${LibtorrentRasterbar_LIBRARIES} ${Boost_LIBRARIES})
+
+    # some flags for cmake 2.6+
+    # much simpler for cmake 3.1+: https://cmake.org/cmake/help/v3.1/module/FindThreads.html
+    if(THREADS_HAVE_PTHREAD_ARG)
+        set_property(TARGET ${sn} PROPERTY COMPILE_OPTIONS "-pthread")
+        set_property(TARGET ${sn} PROPERTY INTERFACE_COMPILE_OPTIONS "-pthread")
```

Comment:
This shouldn't be triggered on Windows (it doesn't have pthreads), though I didn't try Windows compilation yet